Hardware/Hardware/CardReader/StripeCardReader/PaymentIntentParameters+Stripe.swift

```diff
@@ -9,7 +9,13 @@ extension Hardware.PaymentIntentParameters {
             return nil
         }
 
-        let returnValue = StripeTerminal.PaymentIntentParameters(amount: self.amount, currency: self.currency)
+        /// The amount of the payment needs to be provided in the currency’s smallest unit.
+        ///https://stripe.dev/stripe-terminal-ios/docs/Classes/SCPPaymentIntentParameters.html#/c:objc(cs)SCPPaymentIntentParameters(py)amount
+        let amountInSmallestUnit = amount * 100
```

Comment:
This will work for most currencies for the time being.  If / when we get to Japanese Yen we will need to change this. For example: https://wordpress.org/support/topic/stripe-krw-and-jpy-currency-issue/

Reply:
Good point. Logged #3948 for it